src/OutputColorizer.Core/OutputColorizer.Core.xproj

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.targets" Condition="'$(VSToolsPath)' != ''" />
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>c86fc95f-b3e1-4175-b669-346c1702cdae</ProjectGuid>
+  </PropertyGroup>
+  <PropertyGroup>
+    <CLIPath>$(MSBuildThisFileDirectory)..\..\tools\dotnetcli\dotnet.exe</CLIPath>
+    <ProjectJson>$(MSBuildThisFileDirectory)project.json</ProjectJson>
+    <LockFile>$(MSBuildThisFileDirectory)project.lock.json</LockFile>
+    <OutputLocation>$(MSBuildThisFileDirectory)..\..\bin\Release\Core</OutputLocation>
```

Comment:
One more thing we can also do later, is to have the path here be something like:
```xml
<OutputLocation>$(MSBuildThisFileDirectory)..\..\bin\$(Configuration)\Core</OutputLocation>
```
so that if you want to have the output in the Debug folder or Release you can configure it.